lib/ohai/plugins/cloud.rb

```diff
@@ -189,7 +189,7 @@ def get_openstack_values
     cloud[:public_hostname] = openstack["public_hostname"]
     cloud[:local_ipv4] = openstack["local_ipv4"]
     cloud[:local_hostname] = openstack["local_hostname"]
-    cloud[:provider] = openstack["provider"]
+    cloud[:provider] = "openstack"
```

Comment:
It could be breaking for some users. Though I understand your argument, I think it's safer to keep it as it was.